.gitlab-ci.yml

```diff
@@ -518,6 +518,7 @@ format:black:
   script:
     - pip install --no-build-isolation -e .[dev]
     - black --check python/
+    - isort --check . --profile black
```

Comment:
Create an own job for this instead of adding it to the black one - also since you added a config, you can remove the `--profile black` option